src/mobile/mobile_ios.go

```diff
@@ -29,6 +26,7 @@ func (nsl MobileLogger) Write(p []byte) (n int, err error) {
 	return len(p), nil
 }
 
+/*
```

Comment:
Another thing to remember to uncomment/fix or else give up on, depending on if apple ever does anything to improve the situation.